content/docs/buildpack-author-guide/create-buildpack/make-app-runnable.md

```diff
@@ -21,7 +22,7 @@ EOL
 # ...
 ```
 
-Your full `ruby-buildpack/bin/build` script should now look like the following:
+Your full `ruby-buildpack/bin/build`<!--+ "{{open}}" +--> script should now look like the following:
```

Comment:
Were you able to test the hugo output with these tags being placed right next to the code block? IIRC I had some difficulties placing a comment tag right after a code block.

Reply:
The `<!--+ "{{open}}" +-->` comments work with hugo, but I had to modify the placement of all `<!--+ "{{execute}}" +-->` comments.